Shapes.java

```diff
@@ -30,15 +33,54 @@ public static void rectangle(int length2,int width2, int areaRectangle){
  System.out.println("The area of the rectangle is " + areaRectangle);
 
 } //close method for rectangle
-public static void circle(double radius, double area){
+public static void circle(int radius, double area){
  Scanner input = new Scanner(System.in);
  System.out.println("What is the radius of the circle?");
- radius = input.nextDouble();
+ radius = input.nextInt();
 
  area = Math.PI * radius * radius;
 
  System.out.println("The area of the circle " + area);
-} // close mthod for circle
- }//close class
+} // close method for circle
 
+  public static void cube(double s, double areaCube){
```

Comment:
Nice job using static methods! Can you think of a way to have these methods have a return value other than `void`?

Reply:
So I should make a non void method? not sure how to do that :O

Reply:
Yeah! Like some of the examples from class, or that other students have posted in slack.